includes/class-move.php

```diff
@@ -10,13 +10,42 @@
 use Activitypub\Activity\Actor;
 use Activitypub\Activity\Activity;
 use Activitypub\Collection\Actors;
+use Activitypub\Model\Blog;
+use Activitypub\Model\User;
 
 /**
  * ActivityPub (Account) Move Class
  *
  * @author Matthias Pfefferle
  */
 class Move {
+
+	/**
+	 * Initialize the Move class.
+	 */
+	public static function init() {
+		/**
+		 * Filter to enable automatically moving Fediverse accounts when the domain changes.
+		 *
+		 * @param bool $domain_moves_enabled Whether domain moves are enabled.
+		 */
+		$domain_moves_enabled = apply_filters( 'activitypub_enable_primary_domain_moves', false );
```

Comment:
@obenland I know that you are not a fan of constants, but this would be the third possibility to enable a feature, aside from advanced settings and constants.

I think we should have a certain level of consistency. I am not arguing for a constant her, maybe we should think about a `Developer Preview` section in the Advanced settings to enable beta features!?!

Reply:
I'd be happy to deprecate feature constants in favor of filters or advanced settings